readme.txt

```diff
@@ -1,64 +1,65 @@
-Tags: WordPress.com, jet pack, comments, contact, gallery, performance, sharing, security, shortcodes, stats, subscriptions, widgets
+Tags: Jetpack, WordPress.com, backup, security, related content, CDN, speed, anti spam, sharing, SEO, video
 Stable tag: 4.5
 Requires at least: 4.6
 Tested up to: 4.7
 
-Increase your traffic, view your stats, speed up your site, and protect yourself from hackers with Jetpack.
+Whatever sort of site you’re running, Jetpack keeps your site secure and helps you increase traffic.
 
 == Description ==
 
-Jetpack simplifies managing WordPress sites by giving you visitor stats, security services, speeding up images, and helping you get more traffic. Jetpack is a free plugin.
+Whatever sort of site you’re running, Jetpack’s features are designed to keep your site secure, elevate your brand, and engage with more readers better.
 
-= Traffic Growth & Insights =
-Jetpack tells you how many visits your site gets and helps you **get more traffic** with tools like Related Posts, Publicize, Enhanced Distribution, and Sharing.
+= Traffic and SEO Tools =
+Traffic is the lifeblood of any website. Jetpack includes simple-to-use features that help you automatically promote, share, and optimize your content for maximum readership.
 
-= Security =
-Jetpack **protects your site** against brute force attacks and unauthorised logins. We also monitor your site for downtime and keep your plugins updated.
+= Security and Backup Services =
+You shouldn’t have to worry about data loss, downtime, and hacking. Jetpack monitors your site, protects it from brute force attacks, secures your logins, scans for malicious code, and backs up all your data.
 
-= Image Performance =
-Jetpack automatically **optimizes and speeds up** images using the global WordPress.com content delivery network. This saves you hosting costs by reducing bandwidth.
+= Writing and Content Creation =
+Jetpack offers a free, high-speed CDN for your images and a variety of tools to customize your site to your heart’s content.
 
-= Centralized Management =
-With Jetpack comes a **centralized dashboard** on WordPress.com. Manage plugins and menus, publish posts, and view enhanced site stats for all of your sites.
+= Discussion and Community =
+Create a connection with your readers and keep them coming back to your site with subscriptions, comments, contact forms, and spam filtering.
 
-= A few more things that our users love =
-Jetpack includes other features that help you customize your site including Custom CSS, Contact Forms, Galleries and Carousels, Notifications and Subscriptions, Configurable Widgets, and [many more](https://jetpack.com/features).
+= Paid Services =
+Most of Jetpack's features and services are free. Jetpack also provides advanced security and backup services, video hosting, site monetization, and advanced SEO tools in three [simple and affordable plans](https://jetpack.com/pricing).

```

Comment:
Can we avoid using "advanced" to describe SEO Tools, like we've done in other areas of the plugin?